test/test_regression.py

```diff
@@ -141,8 +140,10 @@ def test_notebook_basics_lesson4():
   # basis orthogonality
   dt  = 1.0/2048.0 # found from surrogate *.dat file
   e_6 = EOBNRv2_sur.basis(5,'orthogonal')
-  nose.tools.assert_almost_equal(np.sum(e_5*np.conj(e_6)) * dt,0.0,places=14)
-  nose.tools.assert_almost_equal(np.sum(e_5*np.conj(e_5)) * dt,1.0,places=14)
+  orthogonal_test = np.sum(e_5*np.conj(e_6)) * dt
+  normalized_test = np.sum(e_5*np.conj(e_5)) * dt
+  assert( np.abs(orthogonal_test) < 1e-14 )
+  assert( np.abs(normalized_test-1.0) < 1e-14)
```

Comment:
I think the idiomatic way to do this would be
```
assert( np.allclose(orthogonal_test, 0, rtol=0, atol=1.e-14) )
```
There is also a numpy testing utility (new to me, so I don't know if this is the standard approach)
```
np.testing.assert_allclose(orthogonal_test, 0, rtol=0, atol=1.e-14)
```